views/jira-configuration.hbs

```diff
@@ -221,128 +273,373 @@
                       </aui-dropdown-menu>
                     </td>
                   </tr>
                 {{/each}}
-              </tbody>
-            </table>
-          </div>
+                </tbody>
+              </table>
+            </div>
           {{/if}}
 
-        {{#each failedConnections}}
-          <div class="jiraConfiguration__failedConnections__container">
-            <div class="jiraConfiguration__failedConnections__errorIcon__container">
-              <span class="jiraConfiguration__failedConnections__errorIcon aui-icon aui-icon-medium aui-iconfont-error">
-                Failed connection error
-              </span>
-            </div>
-            <div class="jiraConfiguration__failedConnections__orgInfo">
-              {{#if orgName}}
-                <p class="jiraConfiguration__failedConnections__header">
-                  There was an error for organization {{ orgName }}
-                </p>
-              {{else}}
-                <p class="jiraConfiguration__failedConnections__header">
-                  There was an error for organization with ID: {{ id }}
+          {{#each gheCloud.failedConnections}}
+            <div class="jiraConfiguration__failedConnections__container">
+              <div class="jiraConfiguration__failedConnections__errorIcon__container">
+                <span
+                    class="jiraConfiguration__failedConnections__errorIcon aui-icon aui-icon-medium aui-iconfont-error">
+                  Failed connection error
+                </span>
+              </div>
+              <div class="jiraConfiguration__failedConnections__orgInfo">
+                {{#if orgName}}
+                  <p class="jiraConfiguration__failedConnections__header">
+                    There was an error for organization {{ orgName }}
+                  </p>
+                {{else}}
+                  <p class="jiraConfiguration__failedConnections__header">
+                    There was an error for organization with ID: {{ id }}
+                  </p>
+                {{/if}}
+                <p class="jiraConfiguration__failedConnections__errorMsg">
+                  {{failedConnectionErrorMsg deleted}} You will need to reconnect.
                 </p>
-              {{/if}}
-              <p class="jiraConfiguration__failedConnections__errorMsg">
-                {{failedConnectionErrorMsg deleted}} You will need to reconnect.
-              </p>
 
-             <button class="jiraConfiguration__failedConnections__dismiss delete-connection-link" data-installation-id="{{ id }}">
-               Dismiss
-             </button>
+                <button class="jiraConfiguration__failedConnections__dismiss delete-connection-link"
+                        data-installation-id="{{ id }}">
+                  Dismiss
+                </button>
+              </div>
+            </div>
+          {{/each}}
+        </div>
+      </div>
+      {{/if}}
+
+      {{#if gheServers.length}}
```

Comment:
Okay, this is getting big, maybe some point of time, we need to split this into separate file. (if possible in handlebars)